events/templates/home.html

```diff
@@ -3,8 +3,8 @@
 {% load crispy_forms_tags %}
   {%block content%}
     <h1 class="">Events</h1>
-    <form class="d-inline" action="{% url 'home'%}" method="post">
-      {% csrf_token %}
+    <form class="d-inline" action="{% url 'home'%}" method="get">
+      <!-- {% csrf_token %} -->
```

Comment:
CSRF tokens are used only in POST requests. The comment line can be removed entirely